h2o-hadoop-3/h2o-cdp7.0-assembly/build.gradle

```diff
@@ -0,0 +1,130 @@
+ext {
+    hadoopVersion = 'cdh6.3.0'
+    hadoopMavenArtifactVersion = '3.0.0-cdh6.3.0'
+    orcSupported = true
+    orcHiveExecVersion = "2.1.1-cdh6.3.0"
+}
+
+apply plugin: 'java'
+apply plugin: 'com.github.johnrengelman.shadow'
+
+description = 'H2O HDFS client shadowjar for Hadoop ' + hadoopVersion
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+configurations {
```

Comment:
do I understand it right that we can remove all this once cdh7 is released and replace it all with the usual apply from: '../assemblyjar.gradle'?

Reply:
yes, that is correct